pkg/tool/oc_test.go

```diff
@@ -2,23 +2,43 @@ package tool
 
 import "testing"
 
-type fakeProcessExecutor struct{}
+type fakeProcessExecutor struct {
+	Output string
+}
 
-var output = `
+var output47 = `
 serverVersion:
   major: "1"
   minor: "20"
 `
 
 func (p fakeProcessExecutor) RunProcessAndCaptureOutput(executable string, execArgs ...interface{}) (string, error) {
-	return output, nil
+	return p.Output, nil
 }
 
-func TestOcVersion(t *testing.T) {
-	fp := fakeProcessExecutor{}
+func TestOcVersion47(t *testing.T) {
+	fp := fakeProcessExecutor{Output: output47}
 	oc := NewOc(fp)
 	version, _ := oc.GetVersion()
 	if version != "4.7.0" {
 		t.Error("Version mismatch", version)
 	}
 }
+
+type fakeProcessExecutor48 struct {
```

Comment:
This type can be removed as it isn't being used.

Reply:
removed it